pkgs/tools/security/sudo/default.nix

```diff
@@ -14,18 +15,24 @@
 
 stdenv.mkDerivation rec {
   pname = "sudo";
-  version = "1.9.10";
+  version = "1.9.11";
 
   src = fetchurl {
     url = "https://www.sudo.ws/dist/${pname}-${version}.tar.gz";
-    sha256 = "sha256-RKFGEJjnx7jmrFl0mcJPsuQ3SMDBOai0lE5X0TSaZPQ=";
+    sha256 = "sha256-tUduMNg8oUc02pNw8iBr6yHIoz/IWlBPuKYdGNizUb4=";
   };
 
   prePatch = ''
     # do not set sticky bit in nix store
     substituteInPlace src/Makefile.in --replace 04755 0755
   '';
 
+  # AArch64 build fix, remove when version > 1.9.11.
+  patches = lib.optional stdenv.isAarch64 (fetchpatch {
```

Comment:
I don't think this needs to be conditional

Reply:
Done. I was very much on the fence regarding whether or not to make it optional.